### ISSUE TITLE
add GL_MAX_GEOMETRY_OUTPUT_VERTICES property

### ISSUE THREAD

**Issue by Romuald78** [moderngl/moderngl#471]
Please could you add the following GL property accessible from the context.info dictionary 👍 

GL_MAX_GEOMETRY_OUTPUT_VERTICES 

that would be great ;)

Thx guys ^^

**Comment by einarf** [comment]
```py
{
    'GL_VENDOR': 'NVIDIA Corporation',
    'GL_RENDERER': 'GeForce RTX 2080 SUPER/PCIe/SSE2',
    'GL_VERSION': '3.3.0 NVIDIA 456.71',
    'GL_MAX_GEOMETRY_INPUT_COMPONENTS': 128,
    'GL_MAX_GEOMETRY_OUTPUT_COMPONENTS': 128,
    'GL_MAX_GEOMETRY_TEXTURE_IMAGE_UNITS': 32,
    'GL_MAX_GEOMETRY_UNIFORM_BLOCKS': 14,
    'GL_MAX_GEOMETRY_UNIFORM_COMPONENTS': 2048,
    'GL_MAX_GEOMETRY_OUTPUT_VERTICES': 1024, # <-------
}
```

**Comment by einarf** [comment]
Fixed in 8b06d24e323d9871c716b218e2f3192611686346
